pkg/trace/config/config_test.go

```diff
@@ -270,3 +270,40 @@ func TestAcquireHostname(t *testing.T) {
 	host, _ := os.Hostname()
 	assert.Equal(t, host, c.Hostname)
 }
+
+func TestNormalizeDDEnvVar(t *testing.T) {
+	assert := assert.New(t)
+
+	for name, envVal := range map[string]string{
```

Comment:
Why not something like this?
```suggestion
	for in, out := range map[string]string{
	    "staGing": "staging",
	    "ALL CAPS": "all_caps",
	    // ...
	} {
	    t.Run("", // ...
```
Not sure if you know, but if you leave the first argument to `(*testing.T).Run` empty, it will auto-name each sub-test using an index. In case of failure you'll be able to find the failing test case based on it (granted not as easily in some situation).